modules/services/window-managers/i3.nix

```diff
@@ -763,9 +763,9 @@ in
       '';
 
       home.activation.reloadI3 = dag.entryAfter [ "linkGeneration" ] ''
-        if [[ -v i3Changed && -v DISPLAY ]]; then
+        if [[ -v i3Changed ]]; then
           echo "Reloading i3"
-          ${cfg.package}/bin/i3-msg reload 1>/dev/null
+          ${cfg.package}/bin/i3-msg -s /var/run/user/$UID/i3/ipc-socket.* reload 1>/dev/null
```

Comment:
I think that instead of `/var/run/user/$UID/i3/ipc-socket.*` this should be `$XDG_RUNTIME_DIR/i3/ipc-socket.*`.

Reply:
Good point! However, that gives me `XDG_RUNTIME_DIR: unbound variable` error.

Reply:
I guess this variable will not be set unless it's the user service. Do you think it's possible to change `home-manager-user` from a system to a user service instead of specifying `User = ` parameter?

Reply:
Ah, good point. Didn't think of that. Perthaps `${XDG_RUNTIME_DIR:-/run/user/$UID}` would be a good compromise?

I'm not sure if it would be possible to have it as a user service, I don't know of a good way to run a user service from a system service.

Reply:
Agreed :)